README.md

```diff
@@ -95,13 +118,9 @@ As you can see, you can mix and match dsc resources with common puppet resources
 All [puppet metaparameters](https://docs.puppetlabs.com/references/latest/metaparameter.html) should also be supported.
 
 ## Limitations
-- DSC Composite resources not yet fully supported.
+- DSC Composite Resources are not yet fully supported.
 - PS Objects like 'PSCredential' as parameters value not yet supported.
 
 ## Notes
-The puppet types are build from the source code of the DSC Resources MOF schema files.
-If you want the build Puppet types for your own custom DSC Resources, read the [README_BUILD](https://github.com/msutter/puppet-dsc/blob/master/README_BUILD.md)
-
-## License
```

Comment:
Or it's possibly more like:

~~~md
Copyright (c) 2014 Marc Sutter, original author
Copyright (c) 2015 Puppet Labs
~~~